movai_core_shared/logger.py

```diff
@@ -7,21 +7,21 @@
    - Dor Marcous (dor@mov.ai) - 2022
 """
 import logging
-import sys
 from datetime import datetime
-import json
-from urllib.parse import urlparse
-import http.client
-import threading
 import requests
 from logging.handlers import TimedRotatingFileHandler
 from movai_core_shared.envvars import (
     MOVAI_LOGFILE_VERBOSITY_LEVEL,
-    MOVAI_HEALTHNODE_VERBOSITY_LEVEL,
+    MOVAI_FLEET_LOGS_VERBOSITY_LEVEL,
     MOVAI_STDOUT_VERBOSITY_LEVEL,
     MOVAI_GENERAL_VERBOSITY_LEVEL,
     LOG_HTTP_HOST,
 )
+try:
+    from movai_core_enterprise import remote_logger
```

Comment:
remote_logger should exist in __init__.py for the enterprise in order to import it like this.

as I see in your branch https://github.com/MOV-AI/movai-core-enterprise/blob/Adding-message-handlers/movai_core_enterprise/__init__.py
it does not exist this way

Reply:
Fixed the import so that the __init__.py update is not necessary